frontend/src/components/customer/ProfileDetailsCard.tsx/CountryData.ts

```diff
@@ -0,0 +1,433 @@
+interface CountryType {
```

Comment:
You do not need this type, TypeScript can infer what is contained in `countriesData` without needing this type. Unless you need to export it, but I doubt so....